Python/thread_nt.h

```diff
@@ -496,3 +496,13 @@ PyThread_tss_get(Py_tss_t *key)
     SetLastError(error);
     return result;
 }
+
+size_t
+_PyThread_preferred_stacksize()
+{
+    /*
+     * Windows default stack size is 1MB, overridable by the linker in the
+     * executable
+     */
+    return 1024 * 1024;
```

Comment:
I'm not comfortable with hardcoded values, especially the comment "overridable by the linker in the executable". Maybe we should get the size written by the linker in this case? We don't really need this function for faulthandler, maybe remove the Windows implementation?

Reply:
Sure - I added it for completeness/documentation, but it's not called. I can remove it.